### PR TITLE
Add New Data to JSON and Update ReadMe

### PR DESCRIPTION
**Pull request by sungysang**
U can msg me if u need to call to discuss

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Salesforce DX Project: Next Steps
+# Salesforce DX Project: Next Stepss
```
Comment (Julian88Tex) [inline README.md:1]:
```suggestion
# Salesforce DX Project: Next Steps
```

why would you add another s to this word?

File: Contact.json [hunk 2]
```diff
@@ -17,6 +17,15 @@
...
             "Phone": "987654321"
+        },
+        {
+            "attributes": {
+                "type": "Contact",
+                "referenceId": "V"
+            },
+            "FirstName": "Wutang",
+            "LastName": "Dizle",
+            "Phone": "111111111"
```
Comment (Julian88Tex) [inline Contact.json:28]: @sungysang fyi we wanted you to add two contacts not just one

File: README.md (hunk 1 above)
Comment (sungysang) [reply README.md:1]: It was in the AC

File: Contact.json (hunk 2 above)
Comment (Julian88Tex) [reply Contact.json:28]: @rachelronci what do you think? you want to be added?

**Review by Julian88Tex — APPROVED** [review]
look's great! love the name Dizle

**Pull request merged**